src/definitions.ts

```diff
@@ -30,8 +30,8 @@ export interface OutputAdapter {
 }
 
 export interface LogMethod {
-    (contextId: string, message: string, data?: unknown): void
-    (message: string, data?: unknown): void
+    (contextId: string, message: string, data?: unknown, forceWriteOn?: boolean): void
```

Comment:
At first view, I think we can use loggerOptions: {enableLogLevel?: boolean}. maybe more clearly for the code. But we can't here (conflict with data in second method. What a pity!

So here, I think instead of use `forceWriteOn`, we call `enableLogLevel` maybe more relevant?

Reply:
yes now the name is forceLogging